src/api/fetch.js

```diff
@@ -69,14 +65,5 @@ export default async function fetch({
       init.body = JSON.stringify(data);
     }
   }
-  const response = await _fetch(url, init);
-
-  if (!response.ok) {
```

Comment:
where do we throw this, then?

Reply:
it should be thrown in the implementation of `request()` in HttpAPI i think. The problem here is that we don't count GET 404's as errors but that doesn't seem like a good default generally